oop/tests/unit/helpers/string.spec.js

```diff
@@ -0,0 +1,34 @@
+import { replaceSpecialCharacters as Function } from "@/helpers/string.js";
```

Comment:
Same comment here.

Reply:
Same response. Would you like me to change those names for all the files, and if so, do you have a preferred naming system instead of "Function" and "Component"? Perhaps "TestFunction" and "TestComponent"?

Reply:
Please remove the alias altogether.

Example:
```javascript
import { replaceSpecialCharacters } from "@/helpers/string.js";
```

Reply:
Fixed in commit da6428c41d0bf8387b54346f91d2ea73438a1dce